content/en/account_management/teams.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Teams
+kind: documentation
+---
+
+## Overview
+Datadog Teams allow groups of users to organize their team assets within Datadog and automatically filter their Datadog-wide experience to prioritize these assets.
+
+Use Teams to link resources such as dashboards, services, monitors, and incidents to a group of users. You can also add team-specific links to Slack channels, Jira boards, GitHub repositories, etc.
+
+Team membership is flexible. Users can join teams, be added by other members, or be added by an administrator. Users can belong to multiple teams.
+
+## Setup
+
+### Navigation
+
+To find the [team directory page][1]:
+
+1. Navigate to **Organization Settings** from your account menu.
+1. Under **Groups**, select **Teams**.
+
+The [team directory page][1] lists all teams within your organization. Use the **Only My Teams** slider in the upper right to toggle the view between your teams and all teams in the organization.
+
+{{< img src="account_management/teams-directory.jpg" alt="Organization Settings page, Teams tab, showing a panel highlighting the Caching team" >}}
+
+### Create team
+
+1. On the [team directory page][1], click **New Team** at the upper right.
+1. Choose a **Team Name**.
+1. The **Handle** populates based on your team name.
+1. Provide an optional **Description**.
+1. Use the drop-down to select team members.
+1. Click **Create**.
+
+### Modify team
+
+1. On the [team directory page][1], click the team you would like to modify. A side panel appears with the team details.
+1. Mouse over the item you wish to modify. A pencil icon appears.
+1. Click the pencil icon. A pop-up window appears.
+1. Make your changes, then click the appropriate button to save your changes.
+
+## Team handle
+
+A team handle links teams to Datadog resources. Team handles appear in search bars and facets in the format `team:<team-handle>` or `teams:<team-handle>`. 
+
+To find a team handle:
+1. Click the team's name in the team directory page. A side panel appears with team details.
+1. Look for the **handle** field at the top of the panel. 
+
+To associate a resource with a defined team, a Team must exist in Datadog with a matching team handle. When you click on a resource associated with a defined team, a small window appears with the team handle and additional information. Defined teams provide additional functionality such as the Team filter below. 
+
+Team handles that aren’t associated with a defined team in Datadog have a tag-like experience. Convert any undefined team handles to defined teams to take advantage of Teams features. 
```

Comment:
I was just a bit confused by this sentence -- what is meant by `have a tag-like experience`? Also, is it possible to create team handles without an associated team? I couldn't find a way to do this in-app.